web/README.md

```diff
@@ -14,6 +20,9 @@ While the `make` commands outlined in the [guides readme](https://github.com/the
 $ cd web
 $ hugo server
 ```
+
+Alternatively, you can use `make server`
```

Comment:
I realized there is one trick that makes it easier: you can run `make -C web server` from the root. Not sure if we want to document this.

Reply:
I am worried about the LCD of contributors, so would avoid.